frontend/src/components/ActivityAssessmentDetails/ActivityAssesmentDetailsStyles.js

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import { Col, Row, Button } from 'react-bootstrap';
+
+export const ContentCol = styled(Col)`
+    padding-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
+`;
+
+export const ActivityTitle = styled.div`
+    width: 100%;
+    text-align: center;
+    padding-top: 10px;
+    padding-bottom: 5px;
+    background-color: var(--dark-blue);
+    color: var(--font-color);
+`;
+
+export const TopInfo = styled(Row)`
+    justify-content: space-around;
+    padding-top: 6px;
+    max-width: 1500px;
+    margin-left: auto;
+    margin-right: auto;
+    color: var(--font-color);
+`;
+
+export const TopInfoCard = styled.div`
+    background-color: var(--dark-blue);
+    width: 740px;
+    margin-top: 5px;
+    padding: 10px;
+    //tak, wiem - ale to chyba najprostszy sposób na zrobienie tego, przy czym nie jest absolutnym koszmarem
+    @media (max-width: 1800px) {
+        width: 1500px;
+    }
+`;
+
+export const UserInfo = styled.div`
+    display: flex;
+    justify-content: space-between;
+    flex-flow: column;
+    padding-left: 10px;
+`;
+
+export const ActivityInfo = styled.p`
+    overflow-y: auto;
+`;
+
+export const AnswerRow = styled(Row)`
+    width: 100%;
+    min-height: 300px;
+    max-height: 500px;
+    background-color: var(--dark-blue);
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 10px;
+`;
+
+export const AnswerContent = styled.p`
+    overflow-y: auto;
+`;
+
+export const AnswerCol = styled(Col)`
+    margin-top: 20px;
+    color: var(--font-color);
+`;
+
+export const RemarksCol = styled(Col)`
+    width: 100%;
+    min-height: 150px;
+    max-height: 300px;
+    background-color: var(--dark-blue);
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 10px;
```

Comment:
Jeżeli `margin-bottom` jest równy 0 to możesz zapisać:
```css
margin: 10px auto 0 auto
```
Jeżeli jest jakiś domyślny > 0 to pomiń ten komentarz